src/vignette/util/query_options.clj

```diff
@@ -12,6 +13,8 @@
 (def query-opts-map {:fill (create-query-opt #"^#[a-g0-9]+$|^\w+$" true true)
                      :format (create-query-opt #"^\w+$")
                      :frame (create-query-opt #"[0-9]+" true true)
+                     :status (create-query-opt #"^\w+$")
+                     :fastlyBypass (create-query-opt uuid-regex)
```

Comment:
What's the purpose of `fastlyBypass`? It's not clear if it's some sort of secret key or if a random string that's passed by the user would work. Also, FWIW we tried to keep vignette as agnostic about its environment as much as we could - we tried not to worry about things like fastly, since those really shouldn't be concerns of the application. I'm pretty sure that's been broken already, but why does vignette now need something from fastly?